### DISCUSSION TITLE
Attribute Examples

### DISCUSSION THREAD

**Discussion by GenerallyDisjointed** [freeplane/freeplane#758] [General]
Hi everyone

I wondered if someone could link a url about how attribute work, or some examples?

Looking on the Freeplane site, I can't seem to find a section for it?

Many thanks

**Comment by macmarrum** [comment]
There's probably little information about attributes to be found in the documentation.

Attributes can be defined for each node. The are displayed in a 2-column table

| Key | Value |
|--|--|
|Name|Mac|
|Username|macmarrum|

The actions related to Attributes can be listed in `Help->Command Search (Ctrl+F1)` – search for `attribute`

Once you have at least one attribute, you can right-click on the table to perform actions (New, Delete, Move up, Move down).
Alternatively, you can manage attributes in `View->Controls->Tool panel` – tab `Attributes`

**Reply by GenerallyDisjointed** [reply]
Thankyou, at the moment I use attributes for any extra URL's for a node, I'm wondering what else I can use them for?

Over the past week I've created my own style/template and every day I find something new, it's such an incredible piece of software
